x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_forms/protections/malware.tsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components';
+import { EuiRadio, EuiSwitch, EuiTitle, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { OS, ProtectionModes, MalwareProtectionOSes } from '../../../../types';
+import { ConfigForm } from '../config_form';
+import { policyConfig } from '../../../../store/policy_details/selectors';
+import { usePolicyDetailsSelector } from '../../policy_hooks';
+import { clone } from '../../../../models/policy_details_config';
+
+const ProtectionRadioGroup = styled.div`
+  display: flex;
+  .policyDetailsProtectionRadio {
+    margin-right: ${props => props.theme.eui.euiSizeXXL};
+  }
+`;
+
+export const MalwareProtections = React.memo(() => {
+  const policyDetailsConfig = usePolicyDetailsSelector(policyConfig);
+  const dispatch = useDispatch();
+  const selected = policyDetailsConfig && policyDetailsConfig.windows.malware.mode;
+  const OSes: MalwareProtectionOSes[] = [OS.windows, OS.mac];
```

Comment:
This value is used in hooks but it will be reallocated on every render call. Please use `useMemo` or move it out of the render function.